code/game/machinery/recharger.dm

```diff
@@ -34,142 +34,153 @@
 		. += {"[span_notice("\The [src] contains:")]
 		[span_notice("- \A [charging].")]"}
 
-	if(!(machine_stat & (NOPOWER|BROKEN)))
+	if(machine_stat & (NOPOWER|BROKEN))
+		return
+	var/status_display_message_shown
```

Comment:
```suggestion
	var/status_display_message_shown = FALSE
```

i prefer explicitly typing it like this so we can have a proper boolean dichotomy rather than a `TRUE`/`null` thingy